skimage/transform/_warps.py

```diff
@@ -402,3 +412,399 @@ def swirl(image, center=None, strength=1, radius=100, rotation=0,
     return warp(image, _swirl_mapping, map_args=warp_args,
                 output_shape=output_shape, order=order, mode=mode, cval=cval,
                 clip=clip, preserve_range=preserve_range)
+
+
+def _stackcopy(a, b):
+    """Copy b into each color layer of a, such that::
```

Comment:
For future PRs, I would recommend that we only do big function moves when we are making no other changes, because GitHub doesn't make it easy to see whether anything's changed during the move.